### DISCUSSION TITLE
API issues

### DISCUSSION THREAD

**Discussion by XiaoYang0218** [infiniflow/ragflow#3028] [General]
在调用聊天API时出现类似这种问题，一般是可能出了什么问题，有什么好的解决方法？
![屏幕截图 2024-10-26 180051](https://github.com/user-attachments/assets/03db30c3-161e-4325-90b4-59a84c47edc5)

**Comment by KevinHuSh** [comment]
Please check this out:
![image](https://github.com/user-attachments/assets/292202db-40a5-4015-be76-93c4e87df0e0)
